testing/scenario_app/tool/run_android_tests_smoke.sh

```diff
@@ -8,11 +8,30 @@
 # why `./testing/scenario_app/run_android_tests.sh` did or did not report
 # failures correctly.
 
-# Run this command and print out the exit code.
-../third_party/dart/tools/sdks/dart-sdk/bin/dart ./testing/scenario_app/bin/android_integration_tests.dart \
-  --adb="../third_party/android_tools/sdk/platform-tools/adb" \
-  --out-dir="../out/android_debug_unopt_arm64" \
-  --smoke-test="dev.flutter.scenarios.EngineLaunchE2ETest"
+ADB="../third_party/android_tools/sdk/platform-tools/adb"
+OUT_DIR="../out/android_debug_unopt_arm64"
+SMOKE_TEST="dev.flutter.scenarios.EngineLaunchE2ETest"
 
-echo "Exit code: $?"
-echo "Done"
+# Optionally skip installation if -s is passed.
+if [ "$1" != "-s" ]; then
```

Comment:
nit: Since we don't have a nice help screen, documenting this at the top in a `usage` section would be helpful.

Reply:
Ah sorry, this is a debug only script that's not actually used (I was just using it for my own local hacking)